src/arguments/arguments.go

```diff
@@ -21,6 +21,7 @@ type ArgumentList struct {
 	PartitionFilter       string `default:"[\"Common\"]" help:"JSON array of partitions to collect"`
 	MaxConcurrentRequests int    `default:"10" help:"Maximum number of requests running concurrently"`
 	ShowVersion           bool   `default:"false" help:"Print build information and exit"`
+	SslInsecureSkipVerify bool   `default:"false" help:"Skip verification of the certificate sent by the host."`
```

Comment:
sadly we do not have a standard to name this argument, looking around it is either:

```
validateCerts
SslInsecureSkipVerify
TLSInsecureSkipVerify
```

Reply:
i'm not sure if `TLSInsecureSkipVerify` -> `tls_insecure_skip_verify` on the SDK , have you checked ?

Reply:
yes!
```
  -tls_insecure_skip_verify
        Skip verification of the certificate sent by the host.

```